### PR TITLE
[Network] `az network vhub create/update`: Add parameters `--allow-b2b-traffic` and `--auto-scale-config`

### PR DESCRIPTION
**Pull request by prototypicalpro**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

* `az network vhub create`
* `az network vhub update`

Related AAZ PR: https://github.com/Azure/aaz/pull/175

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Network

**Comment by necusjz** [comment]
@prototypicalpro These two parameters seem too long, please fix it.

**Comment by prototypicalpro** [comment]
> @prototypicalpro These two parameters seem too long, please fix it.

Updated to `--allow-b2b-traffic` and `--auto-scale-config`.

**Comment by prototypicalpro** [comment]
> @prototypicalpro These two parameters seem too long, please fix it.

Updated to `--allow-b2b-traffic` and `--auto-scale-config`.

File: src/virtual-wan/azext_vwan/aaz/latest/network/vhub/_create.py [hunk 1]
```diff
@@ -52,54 +52,84 @@ def _build_arguments_schema(cls, *args, **kwargs):
...
         _args_schema.asn = AAZIntArg(
             options=["--asn"],
+            arg_group="Properties",
             help="VirtualRouter ASN.",
             fmt=AAZIntArgFormat(
                 maximum=4294967295,
                 minimum=0,
             ),
         )
-        _args_schema.vwan = AAZResourceIdArg(
```
Comment (prototypicalpro) [inline src/virtual-wan/azext_vwan/aaz/latest/network/vhub/_create.py:85]: @necusjz I'm using autogeneration with aaz-dev, and it's converting this argument from resource id to string. Is there a way I can fix this in the aaz model, or do I need to modify the generated output? To be clear the [rest api type](https://learn.microsoft.com/en-us/rest/api/virtualwan/virtual-hubs/create-or-update?tabs=HTTP) is a subresource so I'm not sure where string type is coming from.

Comment (necusjz) [reply src/virtual-wan/azext_vwan/aaz/latest/network/vhub/_create.py:85]: `.to_serialized_data()` could be helpful in this case.

**Comment by prototypicalpro** [comment]
@necusjz I've fixed the integration tests and this PR is ready for review.

**Review by necusjz — APPROVED** [review]

**Pull request merged**